contributors.html

```diff
@@ -1,117 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Contributors</title>
-</head>
-<body>
-    <h1>
-        HacktoberFest 2021 Contributors of this repository
-    </h1>
+  </head>
+  <body>
+    <h1>HacktoberFest 2021 Contributors of this repository</h1>
 
     <ul>
-        <li>
-            Mahesh Jain
-        </li>
-        
-        <li>
-            Dikshant Rajput
-        </li>
-        
-        <li>
-            Danish
-        </li>
```

Comment:
Don't change the styling of current file. Please revert the changes in this file and don't format this file with prettier